src/Engines/TypesenseEngine.php

```diff
@@ -522,30 +515,6 @@ protected function usesSoftDelete($model): bool
         return in_array(SoftDeletes::class, class_uses_recursive($model), true);
     }
 
-    /**
-     * Set the search options provided by user.
-     *
-     * @param  array  $options
-     * @return $this
-     */
-    public function withSearchParameters(array $options): static
-    {
-        return $this->setSearchParameters($options);
-    }
-
-    /**
-     * Set the search options provided by user.
-     *
-     * @param  array  $options
-     * @return $this
-     */
-    public function setSearchParameters(array $options): static
-    {
-        $this->searchParameters = $options;
-
-        return $this;
-    }
```

Comment:
Removing these methods would imply a breaking change. Are you sure this is the right call? Everyone who is already using Typesense in Scout now will see their implementation break. But we only launched recently so there might still be a window for this.

Reply:
The thing is, that these methods are inside the Engine and not accessible from the Builder instance. I can leave them inside Engine, just to not introduce any breaking changes.